grpc/src/client/transport/registry.rs

```diff
@@ -26,20 +25,20 @@ impl std::fmt::Debug for TransportRegistry {
 
 impl TransportRegistry {
     /// Construct an empty name resolver registry.
-    pub fn new() -> Self {
+    pub(crate) fn new() -> Self {
```

Comment:
Should we also implement `Default` trait for this type by inheriting it?

Reply:
There is an explicit `Default` implementation below. I removed it and used the `derive` macro instead.